### PR TITLE
Fix delete order caps

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

[ET-2182]

### 🗒️ Description

Fixes delete order caps

### 🎥 Artifacts 
image

### ✔️ Checklist
- [ ] Ran `npm run changelog` to add changelog file(s).
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[ET-2182]: https://stellarwp.atlassian.net/browse/ET-2182?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by stratease** [comment]
Any tests for this?

File: tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php [hunk 1]
```diff
@@ -75,7 +75,27 @@ public function it_should_match_order_actions() {
 			ob_start();
 			$singular_page->render_actions( $order );
 			$html[] = str_replace( $order->ID, '{{order_id}}', ob_get_clean() );
+		}
+
+		$this->assertMatchesHtmlSnapshot( implode( PHP_EOL . '<!--NEXT ITEM-->' . PHP_EOL, $html ) );
```
Comment (dpanta94) [inline tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php:80]: crap :facepalm: thank you!

Comment (dpanta94) [reply tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php:80]: thats the reason i though the change was covered :(

**Review by dpanta94 — COMMENTED** [review]
Looks good, can't approve it since I opened it but feel free to approve and merge!

File: tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php (hunk 1 above)
Comment (stratease) [reply tests/integration/TEC/Tickets/Commerce/Admin/Singular_Order_PageTest.php:80]: Yeah, no problem.

**Review by stratease — APPROVED** [review]

**Pull request merged**